source/features/hide-newsfeed-noise.css

```diff
@@ -9,15 +9,14 @@ published a release
 edited a gollum (wiki)
 */
 
-/* TODO: Revert `classes` #6072 */
-.rgh-hide-newsfeed-noise .news :is(
-[classes~='push'],
-[classes~='fork'],
-[classes~='delete'],
-[classes~='follow'],
-[classes~='release'],
-[classes~='gollum']
-),
+/* TODO: Drop old classes, no longer used */
 .rgh-hide-newsfeed-noise .news :is(.push, .fork, .delete, .follow, .release, .gollum) {
```

Comment:
It seems that some classes like `watch_started` and `follow` are still used, but I don't have other events.